src/sentry/conf/server.py

```diff
@@ -137,19 +137,25 @@ def env(key, default="", type=None):
 
 sys.path.insert(0, os.path.normpath(os.path.join(PROJECT_ROOT, os.pardir)))
 
+base_db = {
+    "ENGINE": "sentry.db.postgres",
+    "NAME": "sentry",
+    "USER": "postgres",
+    "PASSWORD": "",
+    "HOST": "127.0.0.1",
+    "PORT": "",
+    "AUTOCOMMIT": True,
+    "ATOMIC_REQUESTS": False,
+}
+
 DATABASES = {
-    "default": {
-        "ENGINE": "sentry.db.postgres",
-        "NAME": "sentry",
-        "USER": "postgres",
-        "PASSWORD": "",
-        "HOST": "127.0.0.1",
-        "PORT": "",
-        "AUTOCOMMIT": True,
-        "ATOMIC_REQUESTS": False,
-    }
+    "default": base_db,
+    "control_silo": {**base_db, **{"NAME": "control_silo"}},
+    "region_silo": base_db,
```

Comment:
Near these two values, let's make an explicit declaration to the tune of
```python
{
    SiloMode.CONTROL: "control_silo",
    SiloMode.REGION: "region_silo",
}
```
so that it isn't necessary to write
```python
list(model_class._meta._ModelSiloLimit__silo_limit.modes)[0]._name_.lower() + "_silo"
```
in various places below.

To avoid the arbitrary `[0]` index, we'd also have to refactor `ModelSiloLimit` to have only one writable mode, rather than a set. This clashes a bit with #38785 (the first time we've actually used multiple modes per decorator, IIRC) but we can hack our way around that.